test-framework/pom.xml

```diff
@@ -55,10 +58,9 @@
             <artifactId>jakarta.annotation-api</artifactId>
         </dependency>
         <dependency>
-            <groupId>org.jboss.spec.javax.xml.bind</groupId>
-            <artifactId>jboss-jaxb-api_2.3_spec</artifactId>
+            <groupId>jakarta.xml.bind</groupId>
+            <artifactId>jakarta.xml.bind-api</artifactId>
```

Comment:
Yes that's exactly what you should do.

Reply:
@ia3andy I wonder if we should add a recipe adjusting these dependencies. Most people won't use them directly but that might be worth it anyway. The ones that got adjusted this way are JAX-RS and JAXB.

Reply:
@gsmet isn't quarkus providing that dependency?